relay-server/src/actors/processor.rs

```diff
@@ -911,9 +911,27 @@ impl EnvelopeProcessor {
 
     /// Remove replays if the feature flag is not enabled
     fn process_replays(&self, state: &mut ProcessEnvelopeState) {
-        let replays_enabled = state.project_state.has_feature(Feature::Replays);
+        let replays_enabled: bool = state.project_state.has_feature(Feature::Replays);
```

Comment:
This type can be inferred by the compiler.

```suggestion
        let replays_enabled = state.project_state.has_feature(Feature::Replays);
```